disputer/disputer.js

```diff
@@ -71,9 +71,9 @@ class Disputer {
         gas: 1500000,
         gasPrice: this.gasEstimator.getCurrentFastPrice()
       };
-      this.logger.info({
+      this.logger.debug({
```

Comment:
Here, for example, before the bot would send out a message saying that the bot was disputing a liquidation and then when it was finished it would send a second message with the tx result. In this PR implementation, this has been condensed to one message at the end. All information embedded in the first message was also added to this.